### ISSUE TITLE
I can not force the body(dead npc) to float in the air (levitating, flying, swimming flags not working)

### ISSUE THREAD

**Issue by PadreWoW** [OregonCore/OregonCore#1424]
**Revision**:
last
**Operating system**:
winserv2008r2
**Current behavior**: 
I can not force the body(dead npc) to float in the air (levitating, flying, swimming flags not working)

**Expected behavior**: 
For example: Colonel Jules should float in the air in quest "The Exorcism of Colonel Jules" 

**Steps to reproduce the problem**:
Just try to complete quest  "The Exorcism of Colonel Jules"  and you will see that Colonel crawling through the air rather than hovering

**Comment by LordUsagi** [comment]
Can confirm. Even with SetCanFly, it doesnt work. Perhaps changing InhabitType will work but still not ideal.

**Comment by LordUsagi** [comment]
Yeah when I change inhabittype it works proper, just a bit of cosmetic issues with the spells. This could be a bigger issue in core.

**Comment by PadreWoW** [comment]
what inhabittype with what flags?

**Comment by LordUsagi** [comment]
1. Which is walking swimming and fly possibility.

**Comment by PadreWoW** [comment]
You change only DB flag? Without any flags in core script (SetCanFly, SetLevitate)?

**Comment by PadreWoW** [comment]
Can not confirm even with me->SetCanFly(true); me->SetLevitate(true);
![1](https://cloud.githubusercontent.com/assets/6802530/16070066/95b81290-32db-11e6-9af4-fb43dc39c6dc.png)
![2](https://cloud.githubusercontent.com/assets/6802530/16070071/9a1e55a6-32db-11e6-847b-228ff731cf11.png)

**Comment by PadreWoW** [comment]
- Creature (Entry: 999096) has wrong value (8) in InhabitType, creature will not correctly walk/swim/fly

**Comment by PadreWoW** [comment]
any news?
